Cargo.toml

```diff
@@ -50,14 +50,15 @@ version = "0.15.0"
 [profile.release]
 # debug = 1 means line charts only, which is minimum needed for good stack traces
 debug = 1
+split-debuginfo = 'packed'
 strip = 'none'
 # Exit process with SIGABRT when any thread panics
 panic = 'abort'
+# Crash on runtime integer overflow
+overflow-checks = true
```

Comment:
Maybe this should be a separate PR?   Doesn't overflow checking have runtime/perf impact?